larray/core/array.py

```diff
@@ -6024,14 +6028,14 @@ def to_excel(self, filepath=None, sheet_name=None, position='A1', overwrite_file
 
             if new_workbook:
                 sheet = wb.sheets[0]
```

Comment:
You also need to rename this variable, because this used to be a different variable and now it has the same name than the argument, which will break the code below.